examples/async-await.rs

```diff
@@ -0,0 +1,18 @@
+use redis::AsyncCommands;
+
+#[tokio::main]
+async fn main() -> redis::RedisResult<()> {
+  let client = redis::Client::open("redis://127.0.0.1/").unwrap();
```

Comment:
4-spaces indent (or just run `rustfmt` on that file)

Reply:
Fixed.